<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLabel" name="labelVersion">
      <property name="text">
       <string>Version:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEditCmakeMinimumVersion">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>3.10</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="comboBoxProjectType">
      <item>
       <property name="text">
        <string>executable</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>library</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelProjectType">
      <property name="text">
       <string>Project type</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButtonCMakeList">
      <property name="text">
       <string>CMakeLists</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineEditVersion">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Ignore this line</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="pushButtonBuild">
      <property name="text">
       <string>Build</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QComboBox" name="comboBoxVersionChoice">
      <item>
       <property name="text">
        <string>No version</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple version</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Version from file</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="lineEditProjectName">
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Tutorial</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelProjectName">
      <property name="text">
       <string>Project name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelCmakeMinimumVersion">
      <property name="text">
       <string>Cmake minimum version:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPlainTextEdit" name="plainTextEditLog">
      <property name="documentTitle">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
